qa/L0_trace/test.sh

```diff
@@ -543,6 +543,21 @@ if [ -f ./global_trace.log.0 ]; then
     RET=1
 fi
 
+SETTINGS="trace_count trace_rate log_frequency"
+
+for SETTING in $SETTINGS; do
+    # Check `out of range` errors
+    rm -f ./curl.out
+    set +e
+    code=`curl -s -w %{http_code} -o ./curl.out -d'{"'${SETTING}'":"10000000000"}' localhost:8000/v2/models/simple/trace/setting`
+    set -e
+    if [ "$code" != "400" ]; then
+        cat ./curl.out
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
```

Comment:
I think @fpetrini15 recently added some [helpers](https://github.com/triton-inference-server/server/blob/639f758de1bafded665668be805dc0772f56b246/qa/L0_trace/test.sh#L648-L649) for this common code, so you can probably use that here, something like this:
```bash
for SETTING in $SETTINGS; do
  update_trace_setting "simple" -d'{"'${SETTING}'":"10000000000"}'
  assert_curl_failure "Expected error on setting large value for ${SETTING}"
done
```

Reply:
Refactored